engine/liveupdate/src/liveupdate.cpp

```diff
@@ -239,6 +239,11 @@ namespace dmLiveUpdate
     Result VerifyManifestReferences(const dmResource::Manifest* manifest)
     {
         assert(g_LiveUpdate.m_ResourceFactory != 0);
+        const dmResource::Manifest* base_manifest = dmResource::GetManifest(g_LiveUpdate.m_ResourceFactory);
+        // The reason the base manifest is zero here, could be that we're running from the editor.
+        // (The editor doesn't bundle the resources required for this step)
+        if (!base_manifest)
```

Comment:
It previously crashed when loading a manifest from an instance running from the editor.